### ISSUE TITLE
Provide uninstall options?

### ISSUE THREAD

**Issue by NagisaEfi** [JHRobotics/softgpu#94]
For example, "Uninstall" button.
This will do uninstall SoftGPU drivers, and revert C:\Windows\System\d3d*.dll to original (backup) version.

**Comment by JHRobotics** [comment]
Good idea, I didn't have time to add it into the current release, but I will add it in the future.
